RevenueCatUI/Templates/V2/Components/Button/ButtonComponentView.swift

```diff
@@ -134,10 +137,14 @@ struct ButtonComponentView: View {
         case .unknown:
             break
         case .webPaywallLink(url: let url, method: let method):
-            openWebPaywallLink(url: url, method: method)
+            self.openWebPaywallLink(url: url, method: method)
         }
     }
 
+    private func openCodeREDemptionSheet() {
+        Purchases.shared.presentCodeRedemptionSheet()
```

Comment:
Not part of this PR, since I see we were already calling `Purchases.shared` when invalidating the customer info when opening a web paywall link... But I do think these methods should ideally belong to the view model, not to the view.